include/boost/beast/websocket/stream.hpp

```diff
@@ -420,7 +413,7 @@ class stream
 
     /// Get the permessage-deflate status
     void
-    get_status(permessage_deflate_status& status);
+    get_status(detail::permessage_deflate_status &status);
```

Comment:
`detail` symbols are not supposed to be in public APIs

Reply:
The 'permessage_deflate_status' structure has been relocated to 'stream.hpp' as @ashtum suggested.